views/index.hbs

```diff
@@ -1,2 +1,16 @@
-<h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
+<div class="container-fluid d-flex flex-column ">
+    <div>
+        <nav class="navbar p-2">
+            <a href=""><h1>{{title}}</h1></a>
```

Comment:
You generally use headings elements as the title of a document/section, here I think applying a class to your `<a>` would be more syntactically correct. Same remark for your use of `<p>` on line 5.
```
<a class="navbar-title" href="/">{{title}}</a>
<a class="navbar-brand navbar-welcome" href="#">Welcome to {{title}}</a>